Automattic/woocommerce-payments
# Use the branch selected when triggering the pre-release workflow

Fixes #6308 

#### Changes proposed in this Pull Request

- Remove the explicit branch ref `develop` when checking out the branch during the pre-release workflow

The workflow will now use whatever branch we selected in the dropdown list when triggering it.

#### Testing instructions

- Trigger the pre-release workflow but select this branch rather than `develop`
- Confirm that the new branch created was based on this branch
- Clean up the side effects of the workflow (branch + tag + GitHub release)

-------------------


**Post merge**
